internal/pkg/describe/status.go

```diff
@@ -42,6 +48,7 @@ type ServiceStatus struct {
 
 	EcsSvc ecsServiceGetter
 	CwSvc  alarmStatusGetter
```

Comment:
Is there a reason why these fields are public but the remaining two is private?

Reply:
no idea but we should fix that along with the other ones in the struct to be private